include/object/structures_32.bf

```diff
@@ -273,31 +273,36 @@ tagged_union lookup_fault lufType {
 
 -- Fault: size = 8 bytes
 block NullFault {
+    padding 32
     padding 60
     field seL4_FaultType 4
 }
 
 block CapFault {
+    padding 32
     field address 32
     field inReceivePhase 1
     padding 27
     field seL4_FaultType 4
 }
 
 block UnknownSyscall {
+    padding 32
     field syscallNumber 32
     padding 28
     field seL4_FaultType 4
```

Comment:
@lsf37, I thought that the bf language could support variable size blocks for tagged enums?

Reply:
I guess that doesn't matter when the storage isn't allocated based on the enum value.

Reply:
> @lsf37, I thought that the bf language could support variable size blocks for tagged enums?

No, all blocks in a union must have the same size currently. There is special code so that the type field can change in size, but the block size overall must be the same.